include/rmm/mr/host/pinned_memory_resource.hpp

```diff
@@ -24,20 +24,30 @@
 
 namespace rmm::mr {
 
-/*
+/**
  * @brief A `host_memory_resource` that uses `cudaMallocHost` to allocate
  * pinned/page-locked host memory.
  *
  * See https://devblogs.nvidia.com/how-optimize-data-transfers-cuda-cc/
  */
 class pinned_memory_resource final : public host_memory_resource {
  public:
-  pinned_memory_resource()                                         = default;
-  ~pinned_memory_resource() override                               = default;
-  pinned_memory_resource(pinned_memory_resource const&)            = default;
-  pinned_memory_resource(pinned_memory_resource&&)                 = default;
+  pinned_memory_resource()                              = default;
+  ~pinned_memory_resource() override                    = default;
+  pinned_memory_resource(pinned_memory_resource const&) = default;  ///< Default copy constructor.
+  pinned_memory_resource(pinned_memory_resource&&)      = default;  ///< Default move constructor.
```

Comment:
I think the other docstrings in this PR did not use a period here.
```suggestion
  pinned_memory_resource(pinned_memory_resource const&) = default;  ///< Default copy constructor
  pinned_memory_resource(pinned_memory_resource&&)      = default;  ///< Default move constructor
```

Reply:
I like no period at the end of non-sentences.

Reply:
Agreed. I generally tried to avoid, but I may have missed a few.